wodles/aws/buckets_s3/aws_bucket.py

```diff
@@ -0,0 +1,1029 @@
+import copy
+import sys
+import botocore
+import json
+import csv
+import gzip
+import zipfile
+import re
+import io
+import zlib
+from os import path
+import operator
+from datetime import datetime
+
+sys.path.insert(0, path.dirname(path.dirname(path.abspath(__file__))))
```

Comment:
The purpose was to avoid import errors when executing `import wazuh_integration` and `import aws_tools` by adding to `sys.path` the `.../wodles/aws` path. It will be reviewed